### ISSUE TITLE
Make Header/Footer Logo Accessible

### ISSUE THREAD

**Issue by sydneywalcoff** [hackforla/expunge-assist#1230]
### Overview
Because the Header and Footer Logos link to the Landing page _dynamically_ and the image doesn't have alt text, Screen Readers don't know what to call these links. 

### Action Items
- [x] add aria-label to link el describing the link as 'Expunge Assist home'
